addons/payment/models/payment_provider.py

```diff
@@ -64,10 +64,12 @@ def _valid_field_parameter(self, field, name):
     redirect_form_view_id = fields.Many2one(
         string="Redirect Form Template", comodel_name='ir.ui.view',
         help="The template rendering a form submitted to redirect the user when making a payment",
+        ondelete="restrict",
         domain=[('type', '=', 'qweb')])
```

Comment:
```suggestion
        domain=[('type', '=', 'qweb')]),
        ondelete="restrict",
```
Oops, forgot about this. This isn't always applied, but we tend to keep the same order for the field parameters. Here is the list I saved because there is no way I can remember that :sweat_smile: :

**Field parameters order**
Model: string, help, readonly, required, states, default, groups, copy
Float: string, digits, help, readonly, required, states, default, groups, copy
Monetary: string, currency_field, help, readonly, required, states, default, groups, copy
Char: string, help, readonly, required, states, default, groups, copy, translate
Selection: string, help, selection, default, readonly, required, states, groups, copy
Many2One: string, comodel_name, help, readonly, required, states, copy, domain, ondelete, default, groups, check_company
One2Many: string, comodel_name, inverse_name, help, readonly, required, states, copy, domain, default, groups, check_company
Many2Many: comodel_name, relation, column1, column2, string, help, readonly, required, states, copy, domain, default, groups, check_company